src/instruments/schemas/instrument.schema.ts

```diff
@@ -13,7 +14,7 @@ export type InstrumentDocument = Instrument & Document;
     getters: true,
   },
 })
-export class Instrument {
+export class Instrument extends QueryableClass {
```

Comment:
Have we updated the instrument output dto to provide the additional fields introduced by `QueryableClass`?